st2actions/packaging/rpm/st2actions-rhel6.spec

```diff
@@ -23,16 +25,16 @@ sed -i -r "s~logs~/var/log/st2~g" conf/*log*.conf
 %install
 
 mkdir -p %{buildroot}/etc/st2actions
-mkdir -p %{buildroot}/usr/local/lib/python2.7/site-packages
+mkdir -p %{buildroot}%{python2_sitelib}
 mkdir -p %{buildroot}/usr/bin
-cp -R st2actions %{buildroot}/usr/local/lib/python2.7/site-packages/
+cp -R st2actions %{buildroot}%{python2_sitelib}/
 cp -R conf/* %{buildroot}/etc/st2actions
 install -m755 bin/st2actionrunner %{buildroot}/usr/bin/st2actionrunner
 install -m755 bin/st2resultstracker %{buildroot}/usr/bin/st2resultstracker
 install -m755 bin/st2notifier %{buildroot}/usr/bin/st2notifier
 %files
 
-/usr/local/lib/python2.7/site-packages/st2actions*
+/usr/lib/python2.7/site-packages/st2actions*
```

Comment:
Just curious - you use `%{python2_sitelib}` macro / variable in the install block above, but not here.

It there a special reason for it (it doesn't work or something else)? :)